Makefile

```diff
@@ -0,0 +1,18 @@
+PWD = $(shell pwd)
+
+all: lint build
+
+node_modules: package.json
+	docker run -it --init --rm -v $(PWD):/code -w /code node npm install
```

Comment:
can you move this to a `bin/npm` too?

Reply:
If this was a node project I would. But since you never actually have to use `npm` in this project I would rather not introduce a confusing `bin` folder.